ui/desktop/tests/e2e/fixtures/authenticateTest.js

```diff
@@ -0,0 +1,28 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+/* eslint-disable no-undef, no-empty-pattern */
+
+const { electronTest } = require('../fixtures/electronTest');
+const LoginPage = require('../pages/loginPage');
+
+exports.authenticateTest = electronTest.extend({
+  // TODO: Should we put these in a config file?
+  clusterUrl: ({}, use) => {
+    use(process.env.BOUNDARY_ADDR ?? 'http://localhost:9200');
+  },
+  username: ({}, use) => {
+    use(process.env.E2E_PASSWORD_ADMIN_LOGIN_NAME ?? 'admin');
+  },
+  password: ({}, use) => {
+    use(process.env.E2E_PASSWORD_ADMIN_PASSWORD ?? 'password');
+  },
+  authedPage: async ({ electronPage, clusterUrl, username, password }, use) => {
```

Comment:
Is there a way to use this [authentication state](https://playwright.dev/docs/auth) to log in once at the beginning of all tests and reuse that state? (not sure if this would work in the desktop client). We do use this in the Admin UI test suite.

Reply:
Unfortunately this doesn't have support in electron, it works great if you're just using a browser but it seems electron support is lacking. 

It was the first thing I tried getting to work but unless I missed something, it doesn't look like it works properly in electron